src/montecarlo/load_mcs_RVs.jl

```diff
@@ -0,0 +1,110 @@
+using DataStructures
+
+"""
+This function loads the FUND input parameters stored in datadir into the syntax
+needed for the @defmcs macro. This script writes to a text file which can then 
+be copied into a @defmcs macro.
+"""
+function load_mcs_RVs(; txt_out = joinpath(@__DIR__, "mcs_RVs.txt"), datadir = joinpath(@__DIR__, "../../data"), string_labels = false)
+
+    files = readdir(datadir)
+    filter!(i->i!="desktop.ini", files)
+    parameters = OrderedDict{Any, Any}(splitext(file)[1] => readdlm(joinpath(datadir,file), ',') for file in files)
+
+    open(txt_out, "w") do f 
+        for (k, v) in parameters
+            val = loadparameter(v, string_labels)
+            if val != nothing
+                write(f, "$k = $val\n")
+            end
+        end 
+    end
+
+end
+
+# Helper function for concatenating strings in loadparameter
+_concat(x,y)=string(x,", ",y)
+
+"""
+Takes the contents of one parameter file and turns it into the string needed for a definition within a @defmcs macro.
+Input 'p' may be a scalar, a 1-D array (defined as two columns in long format), or a 2-D array (defined as three columns in long format).
+If p contains no distributional values, then this returns nothing, because it will not be a random variable for mcs.
+"""
```

Comment:
Again, macros allow you to directly generate julia code, rather than generating strings representing julia code. But if it's only used once...